src/common.jl

```diff
@@ -85,17 +97,34 @@ abstract type Distribution{F<:VariateForm,S<:ValueSupport} <: Sampleable{F,S} en
 const UnivariateDistribution{S<:ValueSupport}   = Distribution{Univariate,S}
 const MultivariateDistribution{S<:ValueSupport} = Distribution{Multivariate,S}
 const MatrixDistribution{S<:ValueSupport}       = Distribution{Matrixvariate,S}
-const NonMatrixDistribution = Union{UnivariateDistribution, MultivariateDistribution}
+const NonMatrixDistribution = Union{UnivariateDistribution,
```

Comment:
I would advise against this.
`const NonX = Union{NonX1, NonX2}` is only valid as long as you only have `X1, X2` as other subtypes.

Reply:
This is existing code - actually this particular thing was just an automatic whitespace fix because it was going over the line length. I'm happy to undo my accidental edit, but I didn't do this...

Reply:
Reverted.